src/main/java/org/apache/jackrabbit/filevault/maven/packaging/VaultMojo.java

```diff
@@ -179,6 +179,13 @@
     @Parameter
     private MavenArchiveConfiguration archive;
 
+    /**
+     * Classifier to add to the generated package. If given, the artifact will be attached
+     * as a supplemental artifact.
```

Comment:
It is currently attached as primary artifact in https://github.com/apache/jackrabbit-filevault-package-maven-plugin/pull/38/files#diff-0318fb590f2d1e8a076a17cfd097b868L480.

As the classifier parameter currently only influences the final name why not change `finalName` in the first place?

Reply:
Indeed, by providing the `finalName` the filename can be modified based on a profile. For example 
```<profile>
            <id>cloud</id>
            <activation>
                <activeByDefault>false</activeByDefault>
            </activation>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.jackrabbit</groupId>
                        <artifactId>filevault-package-maven-plugin</artifactId>
                        <configuration>
                            <finalName>${project.build.finalName}-cloud</finalName>
                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>
```
The problem with this approach is that other plugins (e.g. install and deploy plugin) are based on the artifactId again and the file get installed to the local repo only with the artifactId again.

Reply:
I don't really get this. You want secondary artifacts for the same module. To make it possible to deploy those those need to be attached as such (with the classifier). I don't see any code for attaching secondary artifacts...